test/testautomation_stdlib.c

```diff
@@ -714,6 +714,15 @@ static int SDLCALL stdlib_getsetenv(void *arg)
 #endif
 #endif
 
+#ifdef SDL_PLATFORM_WINDOWS
+#define SDLTEST_PRIs64 "%I64d"
+#define SDLTEST_PRIu64 "%I64u"
+SDL_COMPILE_TIME_ASSERT(longlong_size,sizeof(long long) == 8);
+#else
+#define SDLTEST_PRIs64 "%lld"
+#define SDLTEST_PRIu64 "%llu"
+#endif
```

Comment:
Why not use SDL_PRIu64?

Reply:
SDL_PRIu64 and SDL_PRIs64 don't include `%`

Reply:
Why do you need them to?
At the very least you could do this:
```c
#define FMT_PRIs64 "%"SDL_PRIs64
#define FMT_PRIu64 "%"SDL_PRIu64
```

Reply:
I can do that. However, (i) there is no 100% guarantee that long long is 64 bits, and (ii) SDL_PRIs64 can most times expand to `"l"` on LP64 systems instead of `"ll"` leading to other -Wformat warnings.

Reply:
Gotcha, then I suggest naming them FMT_PRILL and otherwise leaving this as-is.

Reply:
OK. To finalize: rename to `FMT_PRILLd` and `FMT_PRILLu` for signed and unsigned?

Reply:
Yep, seems okay. We may actually want to put this in SDL_stdinc.h (along with single long versions) so other people can use them too.

Reply:
Yep